src/index.html

```diff
@@ -7,12 +8,68 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute in the link tag points to an SCSS file (`./styles/index.scss`). Ensure that the Parcel bundler is set up to handle SCSS files correctly, as browsers do not natively understand SCSS. If Parcel is configured properly, this is acceptable.